docs/developer-docs/latest/developer-resources/data-management.md

```diff
@@ -411,26 +411,27 @@ To initiate a data transfer:
     <code-block title="YARN">
 
     ```bash
-    yarn strapi transfer --to <destination URL> --to-token <Transfer token>
+    yarn strapi transfer --to <destination URL>
     ```
 
     </code-block>
 
     <code-block title="NPM">
 
     ```bash
-    npm strapi yarn strapi transfer -- --to <destination URL> --to-token <Transfer token>
+    npm strapi yarn strapi transfer -- --to <destination URL>
     ```
 
     </code-block>
     </code-group>
 
-4. Answer **Yes** or **No** to the CLI prompt: "The transfer will delete all data in the remote database and media files. Are you sure you want to proceed?"
-5. If you answer **Yes** in step 4, the transfer operation initiates and, when completed, prints a summary of the transferred types in the CLI.
+4. Add the Transfer token when prompted to do so.
+5. Answer **Yes** or **No** to the CLI prompt: "The transfer will delete all data in the remote database and media files. Are you sure you want to proceed?"
+6. If you answer **Yes** in step 4, the transfer operation initiates and, when completed, prints a summary of the transferred types in the CLI.
 
 ### Bypass all `transfer` command line prompts
 
-When using the `strapi transfer` command, you are required to confirm that the transfer will delete the existing database contents. The `--force` flag allows you to bypass this prompt. This option is particularly useful for implementing `strapi transfer` programmatically.
+When using the `strapi transfer` command, you are required to confirm that the transfer will delete the existing database contents. The `--force` flag allows you to bypass this prompt. This option is particularly useful for implementing `strapi transfer` programmatically. You must pass the `to-token` option with the Transfer token if you use the --force option.
```

Comment:
Since we removed `--to-token` from all examples that included --to-token, what do you think about adding an example here, in between parentheses, for instance? Otherwise, I fear it wouldn't be 100% clear how to use it.